go.mod

```
@@ -19,7 +19,7 @@ require (
 	github.com/dustin/go-humanize v1.0.1
 	github.com/filanov/stateswitch v1.0.1-0.20221122134945-bfa198e3a83a
 	github.com/go-gormigrate/gormigrate/v2 v2.1.2
-	github.com/go-jose/go-jose v2.6.3+incompatible
+	github.com/go-jose/go-jose/v4 v4.0.0-00010101000000-000000000000
```

For this one, how come you can't change it directly?

```suggestion
	github.com/go-jose/go-jose/v4 v4.0.5
```

---

thanks, changed

---

Nice!